contracts/modules/capital/Ramm.sol

```diff
@@ -538,22 +536,23 @@ contract Ramm is IRamm, MasterAwareV2, ReentrancyGuard {
   }
 
   function getInitialObservations(
-    State memory initialState,
+    uint initialPriceA,
+    uint initialPriceB,
     uint timestamp
   ) public pure returns (Observation[3] memory initialObservations) {
 
     uint priceCumulativeAbove;
     uint priceCumulativeBelow;
     uint endIdx = timestamp.divCeil(PERIOD_SIZE);
-    uint previousTimestamp = (endIdx - 11) * PERIOD_SIZE; // 9 days | 1 day | until the update
+    uint previousTimestamp = (endIdx - 11) * PERIOD_SIZE; // 27 days | 3 days | until the deployments
 
     for (uint idx = endIdx - 2; idx <= endIdx; idx++) {
       uint observationTimestamp = Math.min(timestamp, idx * PERIOD_SIZE);
       uint observationIndex = idx % GRANULARITY;
       uint timeElapsed = observationTimestamp - previousTimestamp;
 
-      priceCumulativeAbove += 1 ether * initialState.eth * timeElapsed / initialState.nxmA / 1e9;
-      priceCumulativeBelow += 1 ether * initialState.eth * timeElapsed / initialState.nxmB / 1e9;
+      priceCumulativeAbove += initialPriceA * timeElapsed / 1e9;
+      priceCumulativeBelow += initialPriceB * timeElapsed / 1e9;
```

Comment:
Are there any tests that assert that the initialized twap price is the one we set?